frontend/src/routes/+layout.svelte

```diff
@@ -16,6 +16,8 @@
   export let data: LayoutData;
   const { page, updated } = getStores();
 
+  const { notifyWarning } = initNotificationService(new NotificationService(writable([])));
```

Comment:
it feels weird for this to be where the value of the notification service is declared, specifically what's passed into the init, the rest is fine.

Reply:
True. There's no reason why we can't specify the context value when we define the context.
I refactored it and added 57e68c7.